plugins/score-card/src/components/ScoreCardTable/ScoreCardTable.tsx

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { useEffect } from 'react';
+import { Table, TableColumn, Progress, Link } from '@backstage/core-components';
+import { useAsync } from 'react-use';
+import { errorApiRef, useApi } from '@backstage/core-plugin-api';
+import { scoreToColorConverter } from '../../helpers/scoreToColorConverter';
+import { Chip } from '@material-ui/core';
+import { getWarningPanel } from '../../helpers/getWarningPanel';
+import { scoringDataApiRef } from '../../api';
+import { SystemScoreExtended } from '../../api/types';
+import { EntityRefLink } from '@backstage/plugin-catalog-react';
+
+const useScoringAllDataLoader = () => {
+  const errorApi = useApi(errorApiRef);
+  const scorigDataApi = useApi(scoringDataApiRef);
+
+  const { error, value, loading } = useAsync(
+    async () => scorigDataApi.getAllScores(),
+    [scorigDataApi],
+  );
+
+  useEffect(() => {
+    if (error) {
+      errorApi.post(error);
+    }
+  }, [error, errorApi]);
+
+  return { loading, value, error };
+};
+
+type ScoreTableProps = {
+  scores: SystemScoreExtended[];
+};
+
+export const ScoreTable = ({ scores }: ScoreTableProps) => {
+  const columns: TableColumn<SystemScoreExtended>[] = [
+    {
+      title: 'Name',
+      field: 'systemEntityName',
+      render: systemScore =>
+        systemScore.catalogEntityName ? (
+          <Link
+            to={`/catalog/${systemScore.catalogEntityName.namespace}/${systemScore.catalogEntityName.kind}/${systemScore.catalogEntityName.name}/score`}
+            data-id={systemScore.systemEntityName}
+          >
+            {systemScore.systemEntityName}
+          </Link>
+        ) : (
+          <>{systemScore.systemEntityName}</>
+        ),
+      // this would be better but it cant link to "/score" <EntityRefLink entityRef={{
+      //   kind: 'System',
+      //   namespace: 'default',
+      //   name: systemScore.systemEntityName,
+      // }} defaultKind={'System'} />
```

Comment:
Some more clean up, perhaps?

Reply:
removed. I believe this is it so I will also resolve the conversation so its not messy here. Thanks for the comment!